README.md

```diff
@@ -3,7 +3,78 @@ Legends of the Wulin Character Sheet
 
 <p align="center">
     <a href="https://circleci.com/docs/1.0/status-badges/">
-        <img src="https://circleci.com/gh/gentilboulet/LotWCS.svg?&style=shield&circle-token=784ac32effdcc6931968ae398ce0c87289181e68"
+        <img src="https://circleci.com/gh/gentilboulet/LotWCS.svg?&style=shield"
              alt="build status">
     </a>
 </p>
+
+This app is designed to help players & wulin sage alike to create character sheets for the role-playing game Legends of the Wulin.
+
+Currently, there are no plans to provide a character database to users.
+Instead, LotWCS will provide an import/export feature for you to store your characters.
+
+# Technical stuff
+Here we will bore you with technical considerations, for developers of LotWCS mainly.
+
+## App Model
+This application uses the standard React-Redux pattern.
+The whole application relies on a Redux Store to store the character information.
+The character evolves from an initial state to the current state through user actions.
+
+The Redux architecture handle state evolutions by applying a list of actions to the current state, returning a new updated state.
+The Redux Store handle the dispatching for actions (from user inputs mainly) to the reducers.
+
+The state is used to feed React Components to display the character.
+React Components are subdivided by responsibility: container Components are there to link the store to render Components.
+
+## File hierarchy
+The src directory is divided by responsibility:
+
+### actions
+Here are the action creators, a lot of helper functions present to generate an input to a specific action understandable by the reducers.
+
+### components
+__This directory regroups "renderer" components, handling only rendering and catching user inputs. __
```

Comment:
you have a [strange behavior](https://github.com/gentilboulet/LotWCS/blob/9320442318ee29f68484a6059692ebd00b1c5810/README.md) on github